lms/envs/common.py

```diff
@@ -2656,6 +2656,7 @@
     # Taiwan is specifically not translated to avoid it being translated as "Taiwan (Province of China)"
     "TW": "Taiwan",
     'XK': _('Kosovo'),
+    "MD": "Moldova",
```

Comment:
Follow the patter for Kosovo above so that "Moldova" can also be translated. That is, use _('  ') around the string.